src/cli.rs

```diff
@@ -37,13 +39,25 @@ pub enum Command {
         /// The directory to format. If not provided, the current directory is used
         directory: Option<PathBuf>,
     },
+    #[cfg(feature = "completion")]
+    /// Generate shell completions for the given shell to stdout
+    Completions {
+        /// The shell to generate completions for
+        #[clap(value_enum)]
+        shell: clap_complete::Shell,
+    },
 }
 
+#[cfg(not(build))]
+static LONG_VERSION: String = String::new();
+
+#[cfg(build)]
```

Comment:
could you pls explain what these `cfg` do?

Reply:
I included `cli.rs` in `build.rs`, but some parts of `cli.rs` is not accessible in build script due to dependency issues. So I added a `rustc-cfg` in `build.rs` to separate build script and actual building.